backend/src/models/cycle.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose')
+
+/**
+ * Holds information about a recruitment cycle; attributes are subject to change.
+ *
+ * term: (for example) SP19, FA20
+ * chapter: (or school) UIUC, GaTech
+ */
+
+const Cycle = new mongoose.Schema({
+  term: String,
```

Comment:
I think these are both required fields and should thus have the `required` flag set to `true`